packages/@aws-cdk/aws-cloudtrail/README.md

```diff
@@ -92,4 +92,33 @@ trail.addLambdaEventSelector(["arn:aws:lambda"]);
 
 // Add an event selector to log data events for the provided Lambda functions.
 trail.addLambdaEventSelector([lambdaFunction.functionArn]);
+```
+
+Using the `Trail.onEvent()` API, a Cloudwatch event rule can be created that gets triggered for
+every event logged in CloudTrail.
+To only use the events that are of interest, either from a particular service, specific account or
+time range, they can be filtered down using the APIs available in `aws-events`. The following code
+filters events for S3 and from a specific AWS account and triggers a lambda function. See [Events Delivered via
```

Comment:
```suggestion
filters events for S3 from a specific AWS account and triggers a lambda function. See [Events delivered via
```

alternatively if we want to be consistent with the docs that it points to, also capitalize `Via`